index.html

```diff
@@ -47,142 +53,157 @@ <h2>Software Developer</h2>
         </div>
     </section>
     <section>
-        <div class="line"></div>
-        <div class="skills-list">
-            <div class="lang-skill">
-                <div class="language">
-                    <h3>Language</h3>
+        <div class="line1"></div>
+        <div class="skillEclips">
+            <div class="skills-list">
+                <div class="lang-skill">
+                    <div class="language">
+                        <h3>Language</h3>
+                        <ul>
+                            <li>Javascript</li>
+                            <li>Ruby</li>
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </div>
+                    <div class="skills">
+                        <h3>Skills</h3>
+                        <ul>
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="ecl-frame">
+                    <div class="frameworks">
+                        <h3>Frameworks</h3>
+                        <ul>
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
+                    <div class="ellipse2"></div>
+                </div>
+            </div>
+            <div class="ellipseDS"></div>
+        </div>
+    </section>
+    <section class="work">
+        <h2 class="work-title">MY RECENT WORK</h2>
```

Comment:
- [ ] please make your work section cards bigger and well centralized, so it looks more like the figma version